lib/service/service.go

```diff
@@ -3206,6 +3233,53 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 			log.Info("Exited.")
 			return nil
 		})
+
+		if listeners.minimalReverseTunnelWebEnabled {
```

Comment:
Maybe check for `listeners.reverseTunnelMux != nil` and ditch the attribute? It's one less variable to get out of sync.